JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/QueryClient.java

```diff
@@ -993,21 +993,47 @@ boolean tableExists() {
         return tableCheck;
     }
 
-    public void setElevation(List<StaticPointSource> pointSources, List<Building> buildings, int simulationSrid) {
+    public List<String> getElevationTableList() {
 
-        String elevationTable = EnvConfig.ELEVATION_TABLE;
+        List<String> elevationTables = new ArrayList<>();
+        String sqlString = " SELECT * FROM pg_catalog.pg_tables WHERE schemaname LIKE 'public' AND tablename SIMILAR TO 'elevation_data_*' ";
```

Comment:
I don't think it's a good idea to enforce the table names in this way, for now you can provide the list of tables as input

Reply:
The table names are no longer enforced in this way in the latest commit. There is a SQL query that retrieves the list of tables containing elevation data.